.github/workflows/deploy.yml

```diff
@@ -121,6 +121,8 @@ jobs:
               --set-string config.sentry.dsn=${{ steps.secrets.outputs.API_SENTRY_DSN }} \
               --set-string config.twitter.apiKey=${{ steps.secrets.outputs.TWITTER_API_KEY }} \
               --set-string config.coinMarketCap.apiKey=${{ steps.secrets.outputs.COIN_MARKET_CAP_API_KEY }} \
+              --set-string config.openApiDisable=false \
```

Comment:
please move this to below --set-string image.tag

Reply:
Done